src/order/locally_finite.lean

```diff
@@ -115,6 +115,22 @@ class locally_finite_order (α : Type*) [preorder α] :=
 (finset_mem_Ioc : ∀ a b x : α, x ∈ finset_Ioc a b ↔ a < x ∧ x ≤ b)
 (finset_mem_Ioo : ∀ a b x : α, x ∈ finset_Ioo a b ↔ a < x ∧ x < b)
 
+/-- A locally finite order top is an order where all intervals bounded above are finite. This is
+almost the same as `locally_finite_order` + `order_top`, the difference being for empty types. -/
+class locally_finite_order_top (α : Type*) [preorder α] :=
+(finset_Ioi : α → finset α)
+(finset_Ici : α → finset α)
+(finset_mem_Ici : ∀ a x : α, x ∈ finset_Ici a ↔ a ≤ x)
+(finset_mem_Ioi : ∀ a x : α, x ∈ finset_Ioi a ↔ a < x)
+
+/-- A locally finite order bot is an order where all intervals bounded below are finite. This is
+almost the same as `locally_finite_order` + `order_bot`, the difference being for empty types. -/
```

Comment:
I think pointing out that that it is implied by the other spelling (with a reference to the instance) would be useful here

Reply:
I changed the prasing. Hope that helps.